.github/workflows/ci.yml

```diff
@@ -0,0 +1,116 @@
+---
+name: Django Rest Framework Gis CI Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Python==${{ matrix.env.python }} | ${{ matrix.env.TOXENV }}
+    runs-on: ubuntu-20.04
+
+    services:
+      postgres:
+        image: postgis/postgis:10-2.5
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_USER: postgres
+          POSTGRES_DB: django_restframework_gis
+        ports: 
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    strategy:
+      fail-fast: false
+      matrix:
+        env:
+          - python: 3.6
+            TOXENV: py36-django22-djangorestframework310
+          - python: 3.7
+            TOXENV: py37-django22-djangorestframework310
+          - python: 3.8
+            TOXENV: py38-django22-djangorestframework310
+          - python: 3.9
+            TOXENV: py39-django22-djangorestframework312
```

Comment:
py39-django22-djangorestframework310

Reply:
Thanks for correcting @dhaval-mehta
Can you please review it again?

Reply:
Thanks for correcting @dhaval-mehta
Can you please review it again?

Reply:
I'm not sure py39 is supported in drf 3.1p

Reply:
though tests are passing

Reply:
Yes. Python 3.9 is not supported in DRF 3.10.

Reply:
I don't think that we need to remove it if all tests are passing.